tle/cogs/handles.py

```diff
@@ -349,6 +350,28 @@ async def set(self, ctx, member: discord.Member, handle: str):
         users = await cf.user.info(handles=[handle])
         await self._set(ctx, member, users[0])
 
+    @handle.command(brief="Refreshing the handles of users")
+    @commands.has_any_role("Admin", "Moderator")
+    async def refresh(self, ctx):
+        await ctx.send("Please wait...")
+        registeredUsers = cf_common.user_db.get_cf_users_for_guild(ctx.guild.id)
+        for user in registeredUsers:
+            discordID = user[0]
+            currHandle = user[1].handle
+            oldURL = 'https://codeforces.com/profile/' + currHandle
+            newHandle = str((requests.get(oldURL).url)).split('/')[-1]
+            if newHandle != currHandle:
+                users = await cf.user.info(handles=[newHandle])
+                try:
+                    cf_common.user_db.set_handle(discordID, ctx.guild.id, newHandle)
+                except db.UniqueConstraintFailed:
+                    raise HandleCogError(
+                        f"The handle `{handle}` is already associated with another user."
+                    )
+                cf_common.user_db.cache_cf_user(users[0])
+        embed = discord_common.embed_success(f"Updated handles for server members.")
```

Comment:
I don't know how long this full loop takes. We should send periodic updates if it takes long, like a 33% and 66% progress indicator while still inside the loop. Or maybe more if it takes longer. WDYT?

Reply:
I estimate the runtime of this command on our server to be nearly 8-11 minutes (since the number of API calls made are nearly equal to the "serverstat" command. With the updated code, it will send messages after updating 33% and 66% handles.